components/CreateVirtualCardsForm.js

```diff
@@ -344,6 +347,16 @@ class CreateVirtualCardsForm extends Component {
     const { submitting, errors, multiEmailsInitialState } = this.state;
     return (
       <Box>
+        <P>
+          <FormattedMessage
+            id="VirtualCard.Limitinfo"
+            defaultMessage="Your account is currently limited to {limit} gift cards / day. If you want to increase that limit, please contact <SupportLink></SupportLink>."
+            values={{
+              SupportLink: I18nSupportLink,
+              limit: 100,
```

Comment:
Shouldn't that be `get(collectiveSettings, 'virtualCardsMaxDailyCount') || 100`, like for the other one?

Reply:
Not sure what I was thinking there